### ISSUE TITLE
z2m-frontend not ingress friendly

### ISSUE THREAD

**Issue by ciotlosm** [nurikk/z2m-frontend#14]
When trying to support https://github.com/Koenkk/zigbee2mqtt/issues/4266 in the add-on, testing fails (see: https://github.com/danielwelch/hassio-zigbee2mqtt/issues/397)

Apparently (Based on my testing so far) it seems the problem is: 
https://github.com/nurikk/z2m-frontend/blob/25e0b15344426c9d88c12446f9a5e0b326d80733/src/api.ts#L194

When using `ingress` you get `WebSocket connection to 'ws://<hassos>:8123/api' failed: Unexpected response code: 404`.

I will test further, but I think you need to move away from ${window.location.host} to something else.

**Comment by nurikk** [comment]
Fixed in latest release

**Comment by ciotlosm** [comment]
@nurikk I didn't get to troubleshoot myself, so I can't be of much help at the moment. I've seen this made it in 0.2.13 which was added to latest `dev` for z2m.
I've merged https://github.com/danielwelch/hassio-zigbee2mqtt/pull/401 that will add support in `edge` so people can test this, as this includes your fix above when rebuilding `edge` add-on. Should be available soon (~15 minutes). I will test when done and close this issue if fixed.
